pkg/config/autodiscover/autodiscover_targets.go

```diff
@@ -64,6 +64,43 @@ func FindTestTarget(labels []configsections.Label, target *configsections.TestTa
 	} else {
 		log.Warnf("an error (%s) occurred when looking for operaters by label", err)
 	}
+
+	target.DeploymentsUnderTest = append(target.DeploymentsUnderTest, FindTestDeployments(labels, target)...)
+}
+
+// FindTestDeployments uses the containers' namespace to get its parent deployment. Filters out non CNF test deployments,
+// currently partner and fs_diff ones.
+func FindTestDeployments(targetLabels []configsections.Label, target *configsections.TestTarget) (deployments []configsections.Deployment) {
```

Comment:
in the example tnf_config.yml
you add those lines : 
```
#   deploymentsUnderTest:
#     - name: test
#       namespace: tnf
#       replicas: 2
```
why not picking the namespaces from the config instead of driving it from containers under test ?
if you do want to always drive it from CUT, then remove the DeploymentsUnderTest section

Reply:
The reason why we drive it from targetPodLabels is to minimize the manual config for autodiscovery. The addition to the config is similar to CUT/PUT section. It should not be used unless label based autodiscovery does not cover it.